package.json

```diff
@@ -54,7 +54,8 @@
   "homepage": "https://github.com/tlivings/enjoi",
   "dependencies": {
     "@hapi/bourne": "^2.0.0",
-    "@hapi/hoek": "^9.0.3"
+    "@hapi/hoek": "^9.0.3",
+    "weak-map": "^1.0.5"
```

Comment:
Why is this necessary?

Reply:
can't keep a map with objects as keys without it

Reply:
WeakMap is already supported without an additional dependency for a while now.

Reply:
ah I'm running on Node 10

Reply:
Should be supported. Supported since 0.12 I think.

Reply:
ah my mistake i read that as Node 12